CMakeLists.txt

```diff
@@ -20,21 +20,9 @@ list(APPEND INCLUDE_DIRECTORIES core/test/include/)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-
 set(CMAKE_MACOSX_RPATH 1)
 
-# SQLCIPHER is used by default
-# TODO: activate SQLCipher for Windows
-# WIP: when activating, windows build fails during configuration
-# step of SQLCipher project (because C compiler's path contains spaces)
-if (NOT MSVC AND NOT MINGW)
-    set(SQLCIPHER TRUE CACHE BOOL "Enable SQLCipher encryption support")
-endif()
-
-if (SQLCIPHER)
-    add_definitions("-DSQLCIPHER")
-    add_definitions("-DSQLITE_HAS_CODEC")
-endif()
+add_definitions("-DSQLITE_HAS_CODEC")
```

Comment:
Please keep the `SQLCIPHER` variable so that we can disable it if we want to.

Reply:
If we want to stop encrypt, all we need to do is remove `add_definitions("-DSQLITE_HAS_CODEC")`
then sqlcipher will act exactly like sqlite.

Reply:
Then just surround it with a proper variable as it was before. I don’t want to have to edit a file while `-D` on the `cmake` command line invocation can do it.

Reply:
If this option is not set, Windows build of sqlcipher is failing.